Langs/041E.lang

```diff
@@ -0,0 +1,64 @@
+[MetaData]
+Language=Thai (ไทย)
+Translator=NxAnywhere
+Version=2.2.5.0
+[Font]
+Extra Large=24
+Large=18
+Medium=10
+Small=8.5
+[Strings]
+Donate=สนับสนุน / บริจาค
+Check for Updates=ตรวจสอบการอัปเดต
+Your Windows 11 Compatibility Results are Below=ผลลัพธ์กับ Windows 11 ของคุณอยู่ด้านล่าง
+Now Reach WhyNotWin11 via https://www.whynotwin11.org/=ตอนนี้ถึง WhyNotWin11 ผ่านทาง https://www.whynotwin11.org/
+Results Based on Currently Known Requirements!=ผลลัพธ์ขึ้นอยู่กับความต้องการที่ทราบในปัจจุบัน!
+Translation by=NxAnywhere
```

Comment:
Don't write your username here, otherwise it will be shown twice in the program UI. You can try experimenting it locally beforehand, if you add new or modify lang file using Notepad: C:\Users\...\AppData\Local\WhyNotWin11\Langs

Reply:
You were faster just a couple of seconds :smile:

Reply:
Okey